pom.xml

```diff
@@ -49,6 +64,15 @@
                 <artifactId>maven-surefire-plugin</artifactId>
                 <version>3.0.0-M5</version>
             </plugin>
+
+            <plugin>
+                <artifactId>maven-release-plugin</artifactId>
+                <version>3.0.0-M1</version>
+                <configuration>
+                    <scmCommentPrefix>[ci skip]</scmCommentPrefix>
```

Comment:
Is the release plugin doing any commits here?

Reply:
Yes, the usual 2 commits:

1. removing the `-SNAPSHOT` suffix from the version and creating a tag on this commit
2. incrementing the patch version and adding the `-SNAPSHOT` suffix as new development version

To not trigger the CI workflow for these 2 automatic commits the commit messages get the `[ci skip]` magic keyword added.

Reply:
But that would mean, we have the second commit on the main branch. I would like to have only released, tagged commits on main.